langchain/src/embeddings/ollama.ts

```diff
@@ -0,0 +1,134 @@
+import { OllamaInput, OllamaRequestParams } from "../util/ollama.js";
+import { Embeddings, EmbeddingsParams } from "./base.js";
+
+type CamelCasedRequestOptions = Omit<OllamaInput, "baseUrl" | "model">;
+
+/**
+ * Interface for OllamaEmbeddings parameters. Extends EmbeddingsParams and
+ * defines additional parameters specific to the OllamaEmbeddings class.
+ */
+interface OllamaEmbeddingsParams extends EmbeddingsParams {
+  /** The Ollama model to use, e.g: "llama2:13b" */
+  model: string;
```

Comment:
Should we have a default here like the others?

Reply:
The llm and chatModel classes default to "llama2" here, so I will do it as well - thank you for all the suggestions, will implement ASAP. Much appreciated as I am new to this project andopen source contributions in general, so excuse any rookie mistakes!

Reply:
Would definitely recommend this as well! `llama2` will work, and we're looking to add other embedding models that might be better defaults later on!

Reply:
done!